src/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -754,6 +786,13 @@ class Telemetry : public PluginBase {
      */
     AngularVelocityBody attitude_angular_velocity_body() const;
 
+    /**
+     * @brief Get the current vfr hud (synchronous).
```

Comment:
What is `vfr`? What is `hud`? We need to either explain acronyms or use something human understandable.

I know HUD means head up display but that's not really what this is about.

Can we come up with a better name for this information?

Reply:
Yes. Is what i was saying. As the information is mixed i was thinking and i didn't know what name to use. I guess this message is used generally for hud, that's why is mixing all this information but what name would you use for a struct of climb, throttle and airspeed?

Reply:
Hm. `FixedwingMetrics`?

Reply:
Mmm, ok maybe better. Anyay i thik you can use some of this data for non fixed wing aircraft although is thought for a fixed wing.